wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -7,24 +7,25 @@ module Cardano.Wallet.API.V1.Errors where
 
 import           Universum
 
+import           Cardano.Wallet.API.V1.Types (SyncPercentage, SyncProgress (..),
+                     V1 (..), mkEstimatedCompletionTime, mkSyncPercentage,
+                     mkSyncThroughput)
 import           Data.Aeson
+import           Data.Aeson.Encoding (pairStr)
+import           Data.Aeson.Types (Value (..), typeMismatch)
+import qualified Data.HashMap.Strict as HMS
 import           Data.List.NonEmpty (NonEmpty ((:|)))
 import           Generics.SOP.TH (deriveGeneric)
 import qualified Network.HTTP.Types as HTTP
-import           Servant
-import           Test.QuickCheck (Arbitrary (..), oneof)
-
 import qualified Pos.Client.Txp.Util as TxError
 import qualified Pos.Core as Core
 import qualified Pos.Crypto.Hashing as Crypto
 import qualified Pos.Data.Attributes as Core
+import           Pos.Util.Util (aesonError)
+import           Servant
+
+import           Test.QuickCheck (Arbitrary (..), oneof)
```

Comment:
This `QuickCheck` import probably belongs where it was before.

Reply:
Agree